plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -134,5 +134,8 @@ export type EntitiesCatalog = {
    *
    * @param entityRef - An entity reference to the root of the tree
    */
-  entityAncestry(entityRef: string): Promise<EntityAncestryResponse>;
+  entityAncestry(
+    entityRef: string,
+    authorizationToken?: string,
```

Comment:
Just noting here that this may have been a place where a common context would have been useful as a more opaque request state holder #8042 . This type of auth context argument will start to permeate layers and interfaces.

But for now, having no context: We had a talk and what you have here looks good but it would be good if you could make this last argument an optional options object instead, with the optional token in it.

```ts
entityAncestry(
    entityRef: string,
    options?: { authorizationToken?: string },
)
```

This at least leaves it more open for the future.

Reply:
Sounds good, will make that change and polish up the rest of the PR!

Reply:
PR now ready for review.